src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -45,7 +45,11 @@ class DefaultStorageKeyAllocator
     // FailSafeContext
     const char * FailSafeContextKey() { return Format("g/fsc"); }
 
-    // Access Control List
+    // Access Control
+    const char * AccessControlExtensionEntry(size_t version, FabricIndex fabric)
+    {
+        return Format("a/%x/1/%x", static_cast<unsigned>(version), static_cast<unsigned>(fabric));
```

Comment:
I don't see anything clearing these things out on RemoveFabric.  Followup is OK to add that, but we should make sure it gets added.